### PR TITLE
Finish integrating fineRTS

### PR DESCRIPTION
**Pull request by pengyuejiang**
emop can now use fineRTS algorithm by specifying a parameter

### CODE REVIEW

File: emop-core/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <dependency>
       <groupId>edu.illinois</groupId>
       <artifactId>starts-core</artifactId>
-      <version>1.4-SNAPSHOT</version>
+      <version>1.6-SNAPSHOT</version>
```
Comment (owolabileg) [inline emop-core/pom.xml:19]: We never released emop on Maven central AFAIK, so why do you need to bump up the version?

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java [hunk 2]
```diff
@@ -33,6 +33,10 @@ public class ImpactedClassMojo extends ImpactedMojo {
     @Parameter(property = "updateChecksums", defaultValue = "true")
     private boolean updateChecksums;
 
+    /** Parameter to determine whether fine RTS should be used. */
+    @Parameter(property = "enableFineRTS", defaultValue = "false")
+    private boolean enableFineRTS;
+
```
Comment (owolabileg) [inline emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:39]: Is this the right place for this parameter? That is, how come fineRTS is being enabled here, but not all the other options: method, hyrts etc? Also, is it currently possible for a user to select multiple CIA options? If so, how do you handle that case?

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java [hunk 3]
```diff
@@ -41,6 +45,8 @@ public class ImpactedClassMojo extends ImpactedMojo {
     private TransitiveClosureOptions closureOption;
 
     public void execute() throws MojoExecutionException {
+        this.fineRTSOn = enableFineRTS;
+        this.saveMRTSOn = enableFineRTS;
```
Comment (owolabileg) [inline emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:49]: again, why is this here but not the other options?

**Review by owolabileg — CHANGES_REQUESTED** [review]
Please address the comments and questions that I left in the PR

File: emop-core/pom.xml (hunk 1 above)
Comment (pengyuejiang) [reply emop-core/pom.xml:19]: This is bumping the version for the `starts-core` dependency. This version of eMOP depends on fineSTARTS, which in the implementation has a version number of 1.6-SNAPSHOT. I also think it is a better decision to use a separate version number for fineSTARTS so that it is easier to identify if we used the right version of STARTS.

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java (hunk 2 above)
Comment (pengyuejiang) [reply emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:39]: This Mojo is associated with class-level RPS. Methods and HyRTS are implemented in very different structures. They each have three diffrent Mojos associated with them, while fineRTS has a very convenient interface, so it would be difficult to merge methods and HyRTS code.

File: emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java (hunk 3 above)
Comment (pengyuejiang) [reply emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java:49]: These are changing parameters inside STARTS before calling `super.execute()`. And these are the only required params to enable the use of fineRTS.
